daemon/algod/api/server/v2/generated/types.go

```diff
@@ -498,6 +498,7 @@ type DryrunResponse struct {
 	// Protocol version is the protocol version Dryrun was operated under.
 	ProtocolVersion string            `json:"protocol-version"`
 	Txns            []DryrunTxnResult `json:"txns"`
+	Cost            uint64            `json:"cost"`
```

Comment:
This is a generated file. Instead of modifying it, you should modify the `DryrunResponse` entry in `daemon/algod/api/algod.oas2.json`, then follow the instructions to regenerate: https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/README.md#updating-the-v2-rest-api

Let me know if you have trouble